### ISSUE TITLE
Charts

### ISSUE THREAD

**Issue by moshenskyDV** [armornetworkdev/armor-explorer#17]
I'd like to add some charts to explorer (for example transactions during time or difficulty during time).But right now I don't see endpoints in JSON RPC API that supposed to be used to this.

I think way to make 100500 requests for each block is completely wrong.
I'd like to ask: is that easy to add something to API from C++ side to get some stats across the whole history? (If so - can I try to do this)

**Comment by armornetworkdev** [comment]
Yes... but no problem, I can write an API in a separate c++ software able to handle different threads at the same time. As soon as it is ready, I will put it on the server.
